src/tokenizer/tokenizer.rs

```diff
@@ -130,7 +130,7 @@ where
     }
 }
 
-pub(crate) fn box_tokenizer<A>(a: A) -> Box<BoxedTokenizer>
+pub fn box_tokenizer<A>(a: A) -> Box<BoxedTokenizer>
```

Comment:
why remove `(crate)`?

Reply:
With this, I get the following error when I compile, 

```rust
error[E0364]: `box_tokenizer` is private, and cannot be re-exported
   --> src/tokenizer/mod.rs:156:43
    |
156 | pub use self::tokenizer::{BoxedTokenizer, box_tokenizer};
```